📝 First beta (#189)
astrolicious/studiocms: www/docs/src/content/docs/start-here/getting-started.mdx

## Creating an Astro Studio account

:::note
Astro Studio is currently in closed beta. You'll need an invite to create an account.

> While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.

Reviewer: ```suggestion
We're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again and will be informing everyone through our [Discord channel](https://chat.astro-studiocms.xyz) when signups for Astro Studio are open again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
```

f that, we aint customer support.

Reviewer 2: > we aint customer support.

That is in fact wrong sir...  i spent many years as IT support.  so dont hate on support.

> While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again.  There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.

Reviewer 3: ```suggestion
While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
```

> :::note
> Currently, Astro Studio is in closed beta, and you'll need an invite to create an account. 
> 
> While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
> :::

Reviewer 4: Also not sure about the second part of the aside. It feels too long as a note. Not sure what to do, but I think the only important part is maybe the PR? Otherwise, I'd just remove it to make this short and clear.
```suggestion
:::note
Astro Studio is currently in closed beta. You'll need an invite to create an account.

While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
:::
```

Check out how to set environment variables in Astro StudioCMS by heading to [Environment Variables](/start-here/environment-variables).

> {/* TODO: Update the link when reference is started to look good */}

Reviewer: Can this todo be removed??????

Reviewer 2: if you add the suggestion....

> {/* TODO: Update the link when reference is started to look good */}

Reviewer 4: ```suggestion
```
